NorthstarDLL/NorthstarDLL.vcxproj

```diff
@@ -116,13 +116,288 @@
     </PostBuildEvent>
   </ItemDefinitionGroup>
   <ItemGroup>
+    <ClInclude Include="..\include\httplib.h" />

```

Comment:
Can all these moves be put into a separate PR?

Reply:
Done. See #337